credentials/generate-revocation-set.py

```diff
@@ -81,41 +68,109 @@ def main(use_main_net_dcld, use_test_net_dcld, use_main_net_http, use_test_net_h
     rest_node_url = PRODUCTION_NODE_URL_REST if production else TEST_NODE_URL_REST
 
     if use_rest:
-        revoked_cert_list = requests.get(f"{rest_node_url}/dcl/pki/revoked-certificates").json()["revokedCertificates"]
+        revocation_point_list = requests.get(f"{rest_node_url}/dcl/pki/revocation-points").json()["PkiRevocationDistributionPoint"]
     else:
         cmdlist = ['config', 'output', 'json']
         subprocess.Popen([dcld] + cmdlist)
 
-        cmdlist = ['query', 'pki', 'all-revoked-x509-certs']
+        cmdlist = ['query', 'pki', 'all-revocation-points']
 
         cmdpipe = subprocess.Popen(use_dcld(dcld, production, cmdlist), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
-        revoked_cert_list = json.loads(cmdpipe.stdout.read())["revokedCertificates"]
+        revocation_point_list = json.loads(cmdpipe.stdout.read())["PkiRevocationDistributionPoint"]
 
     revocation_set = []
 
-    for revoked_cert in revoked_cert_list:
-        serialnumber_list = []
+    for revocation_point in revocation_point_list:
+        # 1. Validate Revocation Type
+        if revocation_point["revocationType"] != 1:
```

Comment:
Suggest unnesting (continue if revocationType == 1) and use enums for 1

Reply:
Agreed. Fixed in commit - 1bf0a64b5105df13e56b58fa5ea457d59b3dc6b7.